Stockwatch/Stockwatch.Model/DbContextFactory.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Design;
+
+namespace Stockwatch.Model
+{
+    public class StockwatchDbContextFactory : IDesignTimeDbContextFactory<StockwatchDbContext>
+    {
+        public StockwatchDbContext CreateDbContext(string[] args)
+        {
+            var builder = new DbContextOptionsBuilder<StockwatchDbContext>();
+            var connectionString = "Data Source=..\\stock_database.db";
```

Comment:
Before deploying decide where database should be, a location common to both worker & ui